packages/commonwealth/client/scripts/helpers/ContractHelpers/NamespaceFactory.ts

```diff
@@ -134,7 +135,7 @@ class NamespaceFactory extends ContractBase {
           name + ' Community Stake',
           stakesId,
           '0x0000000000000000000000000000000000000000',
-          16000,
+          100000000,
```

Comment:
When testing, the gas estimate seemed high. I don't know if this is related.
`$201.82
0.07125001ETH`

In any case, why is this different from sepolia?

Reply:
Which transaction was this for. deploy namespace, community stake, or buy/sell?

Reply:
namespace. no eth held, so probably the reason:
<img width="349" alt="Screenshot 2024-02-15 at 2 00 04 PM" src="https://github.com/hicommonwealth/commonwealth/assets/9438198/7e56a6aa-4894-43f7-ac20-e3383aa14508">